Makefile

```diff
@@ -28,6 +28,10 @@ dev: dist
 	GOARCH=$(ARCH) GOOS=$(OS) go build -ldflags "$(LD_FLAGS)" -o $(BIN)
 .PHONY: dev
 
+fips-dev: dist
```

Comment:
What do you think about renaming this target `dev-fips`? To me it feels a bit more consistent with the other target naming... `make dev-fips` feels a bit more natural to me than `make fips-dev`. Below we have `make docker-fips`, which is consistent with this.

```shell
$ make dev-fips
$ make docker-fips
```

Not a blocker at all. I'll leave it up to you.

Reply:
done